lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -513,16 +514,19 @@
 import time
 import operator
 from ansible.module_utils.docker_common import (
-    DockerBaseClass,
     AnsibleDockerClient,
     DifferenceTracker,
+    DockerBaseClass,
+    docker_version,
```

Comment:
You don't seem to use `docker_version` anymore (you changed to `self.client.docker_py_version`, which is better anyway :) ).

Reply:
👍